$NetBSD: patch-Cargo.toml,v 1.1 2024/06/08 18:51:10 wiz Exp $

Depends on an unpublished crate. Can't use cargo.mk to fetch it.

--- Cargo.toml.orig	1970-01-01 00:00:00.000000000 +0000
+++ Cargo.toml
@@ -81,7 +81,7 @@ libc = { version = "0.2.153" }
 libcst = { version = "1.1.0", default-features = false }
 log = { version = "0.4.17" }
 lsp-server = { version = "0.7.6" }
-lsp-types = { git = "https://github.com/astral-sh/lsp-types.git", rev = "3512a9f", features = [
+lsp-types = { path = "lsp-types-3512a9f33eadc5402cfab1b8f7340824c8ca1439", features = [
   "proposed",
 ] }
 matchit = { version = "0.8.1" }
@@ -104,7 +104,7 @@ rand = { version = "0.8.5" }
 rayon = { version = "1.10.0" }
 regex = { version = "1.10.2" }
 rustc-hash = { version = "1.1.0" }
-salsa = { git = "https://github.com/salsa-rs/salsa.git", package = "salsa-2022", rev = "05b4e3ebdcdc47730cdd359e7e97fb2470527279" }
+salsa = { path = "salsa-05b4e3ebdcdc47730cdd359e7e97fb2470527279", package = "salsa-2022" }
 schemars = { version = "0.8.16" }
 seahash = { version = "4.1.0" }
 serde = { version = "1.0.197", features = ["derive"] }
